XMOJ.user.js

```diff
@@ -1741,14 +1741,16 @@ else {
                                     UsernameLink.className = "link-primary link-offset-2 link-underline-opacity-50";
                                     if (UtilityEnabled("Rating")) {
                                         let rating = await GetRating(RowData.Username).then();
-                                        if (rating > 500) {
+                                        if (RowData.QuickSubmitCount >= 2) {
+                                            UsernameLink.className += " link-info";
+                                        } else if (rating > 500) {
                                             UsernameLink.className += " link-danger";
                                         } else if (rating >= 400) {
                                             UsernameLink.className += " link-warning";
                                         } else if (rating >= 300) {
-                                            UsernameLink.className += " link-success"
```

Comment:
I have no idea why there wasn't a ; here

Reply:
Maybe a forgot it.

Reply:
Please add it for me, thanks!